dashboard/app/views/levels/editors/fields/_validation_code.html.haml

```diff
@@ -58,3 +65,8 @@
         complete. If this box is unchecked, hitting 'Finish' will mark the
         level as complete.
 
+:ruby
+  sample_validation_functions = {
+    map: GamelabJr::SAMPLE_VALIDATION_FUNCTIONS.to_json
+  }
+%script{src: webpack_asset_path('js/levels/editors/fields/_validation_code.js'), data: sample_validation_functions}
```

Comment:
I just noticed, there is already a script tag for this js file earlier in this file. to fix this, and also tidy up a bit, I would recommend:
* inline `sample_validation_functions` and get rid of the :ruby block
* remove this script tag and move the data you are passing in up to the script tag at the top of the file

so I think you can remove lines 68-72 here and have line 2 become:
```
%script{src: webpack_asset_path('js/levels/editors/fields/_validation_code.js'), data: {map: GamelabJr::SAMPLE_VALIDATION_FUNCTIONS.to_json}}
```
as we both learned, you can apparently wrap long haml lines onto the next line when `{}` braces are involved, so feel free to give that a try, too:
```
%script{src: webpack_asset_path('js/levels/editors/fields/_validation_code.js'), 
        data: {map: GamelabJr::SAMPLE_VALIDATION_FUNCTIONS.to_json}}
```